_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -0,0 +1,224 @@
+just be more laid back.-->{% endcomment %}
+
+- **Minsc adds new features:** a recent [release][minsc 0.2] of this
+  policy language and compiler adds support for several new data types:
+  pubkeys, hashes, polices, [miniscript][topic miniscript],
+  [descriptors][topic descriptors], addresses, and networks.  It also
+  adds new functions for conversions: `miniscript()`, `wsh()`, `wpkh()`,
+  `sh()` and `address()`.  Here's an [example][minsc example] taken from
+  the project's website that uses a specified public key, as a [BIP32][]
+  extended public key with derivation path, and several conversion
+  functions, returning the set of data specified on the final line:
+
+    ```hack
+    $alice = xpub68Gmy5EdvgibQVfPdqkBBCHxA5htiqg55crXYuXoQRKfDBFA1WEjWgP6LHhwBZeNK1VTsfTFUHCdrfp1bgwQ9xv5ski8PX9rL2dZXvgGDnw/9/0;
+
+    $policy = pk($alice/1/3) && older(1 month);
+    $miniscript = miniscript($policy); // compile policy to miniscript
+    $descriptor = wsh($miniscript); // wrap with a p2wsh descriptor
+    $address = address($descriptor); // generate the address
+
+    [ $policy, $miniscript, $descriptor, $address ]
+    ```
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Periodically make block-relay connections and sync headers][review club
+#19858] is a PR ([#19858][Bitcoin Core #19858]) by Suhas Daftuar to make [eclipse
+attacks][topic eclipse attacks] more difficult.  The proposal initiates
+outbound connections every 5 minutes on
+average to sync headers and potentially learn of new blocks, in which case it
+rotates out an existing block-relay-only peer in favor of the new peer. This
+would raise the cost to an attacker to sustain a partitioning attack against a
+node, because as long as the node has at least one honest peer in its addrman,
+it should always eventually find the honest chain with the most work.
+
+Most of the discussion focused on understanding the proposed change.
+
+{% include functions/details-list.md
+  q0="What are block-relay-only connections?"
+  a0="Block-relay-only connections are a type of connection introduced in
+      [Bitcoin Core #15759][Bitcoin Core #15759] that does not participate in
+      transaction or address relay---it only relays blocks."
+
+  q1="Are block-relay-only connections less observable than full-relay
+      connections?"
+  a1="Yes, full-relay connections gossip transactions and addresses, which can
+      reveal information about the network topology and be used by a spy to map
+      connections."
+  a1link="https://bitcoincore.reviews/19858#l-59"
+
+  q2="Describe a scenario in which periodic block-relay-only connections could
+      help prevent an attack?"
+  a2="When a node has been eclipsed but its node address manager (addrman) still has honest addresses
```

Comment:
Yeah, I don't believe I've seen or heard anyone write or say "node address manager" but I don't know how familiar our readers are with the term "addrman".